hawc/apps/eco/views.py

```diff
@@ -76,6 +76,50 @@ def get_success_url(self):
 # Term preview
 class NestedTermList(ListView):
     model = models.NestedTerm
+    paginate_by = 2000
+    filterset_class = filterset.NestedTermFilterset
+
+    def include_ancestors(self, queryset=None):
```

Comment:
This is brilliant. I'm going to refactor a little bit but this approach is perfect; I thought we were going to have hundreds of queries but this puts it all into one; nice job